containers/proxy/config/conf.d/tracing.conf

```diff
@@ -1,8 +1,14 @@
 # this creates a tracing ID that will be propagated to all following requests.
 # it will use the current RequestID but prefer the value from an incoming X-Tracing-ID header.
 # this allows re-using the tracingID for sub requests made by an application.
-# example: user->proxy->webserver->php (Uvex TYPO3)->proxy->webserver->php (Uvex Shopware) will use the same tracingID
+# example: user->proxy->webserver->php->proxy->solr will use the same tracingID
 map $http_x_tracing_id $traceId {
-    default   $http_x_tracing_id;
-    ""        $request_id;
-}
\ No newline at end of file
+	# the incoming header is empty (or does not exist) -> use current requestID
+	"" $request_id;
+
+	# the incoming header is a hex string -> use it
+	"~^[0-9a-fA-F]+$" $http_x_tracing_id;
```

Comment:
Do we have any technical restrictions to limit this to HEX? To be able to use this as a debugging tool I could imagine to allow alpha numeric and dash.

I would add a length restriction (e.g. `{8,32}`) to the header